packages/core/test/datasources/Datasource-test.js

```diff
@@ -308,11 +309,13 @@ describe('Datasource', function () {
       var result = datasource.select({ features: { custom: true } }, done), quads = [];
       result.on('data', function (q) { quads.push(q); });
       result.on('end', function () {
-        quads.should.deep.equal([
-          { subject: 's', predicate: 'p', object: 'o1', graph: 'http://example.org/#mygraph' },
-          { subject: 's', predicate: 'p', object: 'o2', graph: 'http://example.org/#mygraph' },
-          { subject: 's', predicate: 'p', object: 'o3', graph: 'g' },
-        ]);
+        let matchingquads = [{ subject: N3.DataFactory.namedNode('s'), predicate: N3.DataFactory.namedNode('p'), object: N3.DataFactory.namedNode('o1'), graph: N3.DataFactory.namedNode('http://example.org/#mygraph') },
+                             { subject: N3.DataFactory.namedNode('s'), predicate: N3.DataFactory.namedNode('p'), object: N3.DataFactory.namedNode('o2'), graph: N3.DataFactory.namedNode('http://example.org/#mygraph') },
+                             { subject: N3.DataFactory.namedNode('s'), predicate: N3.DataFactory.namedNode('p'), object: N3.DataFactory.namedNode('o3'), graph: N3.DataFactory.namedNode('g') }];
+        for (let i = 0; i < quads.length; i++)          {
```

Comment:
This check will produce a false positive if quads.length !== matchingQuads.length.